src/lib/apputils/net-server.c

```diff
@@ -889,9 +893,15 @@ setup_addresses(struct socksetup *data)
                                  _("Failed setting up a %s socket (for %s)"),
                                  bind_type_names[addr.type],
                                  paddr(ai->ai_addr));
-                goto cleanup;
+                if (ret != EAFNOSUPPORT)
```

Comment:
You're right, it would rather complicate things to not log at all.  One alternative could be to log a different message for EAFNOTSUPPORT when it is encountered. That said, though, `loop_setup_network()` takes care of logging that there was a fatal error in setup, so I could live with the code as written.